docker-compose.yml

```diff
@@ -81,18 +90,18 @@ services:
         reservations:
           memory: 512M
 
-  testrpc:
-    image: trufflesuite/ganache-cli
-    ports:
-      - "8545:8545"
-    command: '-a 1 --seed 2 --host 0.0.0.0 --mnemonic "${TEST_MNEMONIC}"'
-    deploy:
-      replicas: 1
-      resources:
-        limits:
-          memory: 1024M
-        reservations:
-          memory: 128M
+#  testrpc:
```

Comment:
Let's restore this in this PR, but we should thing about removing it from the dockerfile for sure